src/steamship/data/workspace.py

```diff
@@ -13,15 +13,15 @@
 from steamship.base.response import Response as SteamshipResponse
 
 
-class Space(CamelModel):
+class Workspace(CamelModel):
```

Comment:
Could you add the missing list static method? Listing workspaces sounds like an important feature.